app/services/search_payload.rb

```diff
@@ -0,0 +1,36 @@
+class SearchPayload
+  attr_reader :registerable_calendar
+  delegate :slug,
+           :title,
+           :description,
+           :state,
+           :need_ids,
+           :paths,
+           :prefixes,
+           :indexable_content,
+           :content_id,
+           to: :registerable_calendar
+
+  def initialize(registerable_calendar)
+    @registerable_calendar = registerable_calendar
+  end
+
+  def self.call(registerable_calendar)
+    new(registerable_calendar).call
+  end
+
+  def call
+    {
```

Comment:
Some of these attributes (`owning_app`, `paths` and more)  are not valid in rummager. Rummager will silently discard them probably, but good to remove anyway.

You can check [field_definitions.json](https://github.com/alphagov/rummager/blob/master/config/schema/field_definitions.json) for all possible attributes, and [document_types/edition.json](https://github.com/alphagov/rummager/blob/master/config/schema/document_types/edition.json) for all field names allowed in an `edition`.